.github/workflows/release-binaries.yaml

```diff
@@ -38,6 +38,8 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           cache: maven
           native-image-job-reports: true
+      - name: GraalVM supported '-march' options
```

Comment:
This is useful output to keep, as there is really no public list of that output: you need to run on the specific hardware to get the latest supported version.

So, when there will be new versions of GraalVM, this will provide us with the latest set of options available.